docs/api/examples.md

```diff
@@ -90,7 +90,31 @@ $ curl -X POST -H "Authorization: Token d2f763479f703d80de0ec15254237bc651f9cdc0
     "id": 16,
     "name": "My New Site",
     "slug": "my-new-site",
-    "region": null,
+    "region": 5,
```

Comment:
The curl command above needs to be extended to include the `region` field.

Reply:
Added region to the curl command and a note stating that region is an example of a non required field.